scripts/pwned-passwords.sh

```diff
@@ -22,28 +22,15 @@ Usage: ${0} [-nufdph]
 EOM
 }
 
-check_7z() {
-  if ! command -v 7z &> /dev/null; then
-    while true; do
-      read -p "7z is not installed. Do you wish to install (y/n)? " yn
-      case $yn in
-        [Yy]* ) brew install p7zip; break ;;
-        [Nn]* ) exit;;
-        * ) echo "Please answer yes or no." ;;
-      esac
-    done
-  fi
-}
-
 download_pwned_passwords() {
   echo "Downloading pwned passwords. This may take awhile ..."
-  curl $pwned_url --output $pwned_7z
+  ./lib/pwned_password_downloader.rb
```

Comment:
I think we may want to `bundle exec` this to make sure it's able to find the right gems?
```suggestion
  bundle exec ./lib/pwned_password_downloader.rb
```